fuzz/client.c

```diff
@@ -50,8 +50,6 @@ int FuzzerInitialize(int *argc, char ***argv)
     ERR_get_state();
     CRYPTO_free_ex_index(0, -1);
     idx = SSL_get_ex_data_X509_STORE_CTX_idx();
-    RAND_add("", 1, ENTROPY_NEEDED);
-    RAND_status();
```

Comment:
See https://github.com/openssl/openssl/pull/4025 which does this the "right" way.